tests/cassettes/TestDownload.test_download_replaces_api_version_in_url.yaml

```diff
@@ -0,0 +1,148 @@
+interactions:
```

Comment:
@ZeldaZach 
Does this need to go with a provider? `uri: https://api.tcgplayer.com/token` or is this called elsewhere?

The entire PR looks like it's only cassettes, I guess you must of had a lot of tests already? What's the current code coverage?